core-bundle/src/Command/MigrateCommand.php

```diff
@@ -362,6 +364,19 @@ private function executeMigrations(bool &$dryRun, bool $asJson, string $specifie
                 // Do not run the update recursive if a hash was specified
                 break;
             }
+
+            if ($loopControl-- < 1) {
+                if ($asJson) {
+                    $this->writeNdjson('error', [
+                        'message' => 'The migrations were stopped after 19 iterations as a precaution. There is a high chance of an infinite loop of migrations.',
+                        'isSuccessful' => false,
+                    ]);
```

Comment:
@aschempp does this work for the manager?

Reply:
as far as I know, error messages will just be shown as is in the UI.